source/usage-examples/deleteMany.txt

```diff
@@ -0,0 +1,72 @@
+=====================
+Delete Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can delete several documents in a collection at once with
+``collection.deleteMany()``.
+The ``deleteMany()`` method uses a query document that you provide
+to match only the subset of the documents in the collection that match
+the query. If you don't provide a query document (or if you provide an
+empty document), MongoDB matches all documents in the collection. All
+matched documents are deleted.
```

Comment:
Interesting. Is there a preferred method for, say, deleting all documents in a collection? That might be better than no query document. http://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#drop ?